meinberlin/apps/kiezradar/api.py

```diff
@@ -1,10 +1,28 @@
 from rest_framework import viewsets
 from rest_framework.permissions import IsAuthenticated
 
+from .models import KiezRadar
 from .models import SearchProfile
+from .serializers import KiezRadarSerializer
 from .serializers import SearchProfileSerializer
 
 
+class KiezRadarViewSet(viewsets.ModelViewSet):
```

Comment:
@goapunk should we use here the a4 mixin for the permissions? maybe the same for the search profile view